[2023-06-26T19:00:00.933+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dbt_ipflow_dag.send_get_requests manual__2023-06-26T18:52:04.674534+00:00 [queued]>
[2023-06-26T19:00:00.941+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dbt_ipflow_dag.send_get_requests manual__2023-06-26T18:52:04.674534+00:00 [queued]>
[2023-06-26T19:00:00.942+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-26T19:00:00.944+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-06-26T19:00:00.945+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-26T19:00:00.956+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): send_get_requests> on 2023-06-26 18:52:04.674534+00:00
[2023-06-26T19:00:00.962+0000] {standard_task_runner.py:55} INFO - Started process 27316 to run task
[2023-06-26T19:00:00.971+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_ipflow_dag', 'send_get_requests', 'manual__2023-06-26T18:52:04.674534+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/dbt_ip_flow_dag.py', '--cfg-path', '/tmp/tmp5w4ye50p']
[2023-06-26T19:00:00.993+0000] {standard_task_runner.py:83} INFO - Job 67: Subtask send_get_requests
[2023-06-26T19:00:01.072+0000] {task_command.py:389} INFO - Running <TaskInstance: dbt_ipflow_dag.send_get_requests manual__2023-06-26T18:52:04.674534+00:00 [running]> on host 635f1a4ac995
[2023-06-26T19:00:01.147+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dbt_ipflow_dag
AIRFLOW_CTX_TASK_ID=send_get_requests
AIRFLOW_CTX_EXECUTION_DATE=2023-06-26T18:52:04.674534+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-26T18:52:04.674534+00:00
[2023-06-26T19:00:01.150+0000] {connection.py:300} INFO - Snowflake Connector for Python Version: 3.0.4, Python Version: 3.7.15, Platform: Linux-5.15.49-linuxkit-aarch64-with-debian-11.5
[2023-06-26T19:00:01.152+0000] {connection.py:1013} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2023-06-26T19:00:01.153+0000] {connection.py:1030} INFO - Setting use_openssl_only mode to False
[2023-06-26T19:00:01.851+0000] {cursor.py:806} INFO - query: [SELECT * FROM DEV_QA.DBT_POC.STG_IPFLOW_INPUT_DATA]
[2023-06-26T19:00:02.390+0000] {cursor.py:819} INFO - query execution done
[2023-06-26T19:00:02.391+0000] {cursor.py:962} INFO - Number of results in first chunk: 1
[2023-06-26T19:00:02.989+0000] {cursor.py:806} INFO - query: [INSERT INTO DEV_IP_FLOW.RAW_DATA.IP_FLOW_API_OUTPUT_DATA (USER_IP, LAST_RESPONSE...]
[2023-06-26T19:00:04.352+0000] {cursor.py:819} INFO - query execution done
[2023-06-26T19:00:04.354+0000] {cursor.py:806} INFO - query: [COMMIT]
[2023-06-26T19:00:04.447+0000] {cursor.py:819} INFO - query execution done
[2023-06-26T19:00:04.449+0000] {cursor.py:962} INFO - Number of results in first chunk: 1
[2023-06-26T19:00:04.451+0000] {connection.py:605} INFO - closed
[2023-06-26T19:00:04.546+0000] {connection.py:608} INFO - No async queries seem to be running, deleting session
[2023-06-26T19:00:04.693+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-26T19:00:04.722+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dbt_ipflow_dag, task_id=send_get_requests, execution_date=20230626T185204, start_date=20230626T190000, end_date=20230626T190004
[2023-06-26T19:00:04.773+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-26T19:00:04.800+0000] {taskinstance.py:2582} INFO - 2 downstream tasks scheduled from follow-on schedule check
